mtools/mlaunch/mlaunch.py

```diff
@@ -676,7 +657,7 @@ def kill(self):
         processes = self._get_processes()
 
         # convert signal to int
-        sig = self.args['signal']
+        sig = self.args.get('signal') or 15
```

Comment:
15 is a magic number .. worth commenting that this is SIGTERM (aka default trigger graceful shutdown)